run_dir/design/deliveries.html

```diff
@@ -23,44 +23,36 @@
 
 <div id="page_content" class="deliveries-page">
   <div class="row">
-    <div>
-      <h1>Ongoing <span id="page_title">Deliveries</span></h1>
-      <h3>Total number of:</h3>
-      <h5>projects: {{ number_of_projects }}</h5>
-      <h5>flowcells: {{ number_of_flowcells }}</h5>
-      <h5>lanes: {{ number_of_lanes }}</h5>
-      <h5>samples: {{ number_of_samples }}</h5>
       <div id="ongoing_deliveries">
-
-        <div class="delivery-filters" style="display:none;">
+        <h1>Ongoing <span id="page_title">Deliveries</span></h1>
+        <h3>Total number of:</h3>
+        <h5>projects: {{ number_of_projects }}</h5>
+        <h5>flowcells: {{ number_of_flowcells }}</h5>
+        <h5>lanes: {{ number_of_lanes }}</h5>
+        <h5>samples: {{ number_of_samples }}</h5>
+        <div class="row delivery-filters"  style="margin-left:30px;">
           <hr>
-          <form class="form-inline deliveries-filters">
-            <div class="form-group">
-              <label for="bioinfo-filter-assigned" class="control-label">Assigned</label>
-              <select class="form-control input-sm" id="bioinfo-filter-assigned">
-                <option>Anyone</option>
-              </select>
-            </div>
-            <div class="form-group">
-              <label for="bioinfo-filter-application" class="control-label">Application</label>
-              <select class="form-control input-sm" id="bioinfo-filter-application">
-                <option>Any</option>
-              </select>
+            <div class="status-filters">
+              <h4>Filter by flowcell status</h4>
+              <div id="visible-classes" class="{{ ' '.join(status_list.keys()) }}"></div>
+              <div id="filtered-classes" class=""></div>
+              {% for status, number in sorted(status_list.items()) %}
+                <label class="checkbox-inline">
```

Comment:
I think these would work better as a vertical list. Maybe have three columns? You can also use a bootstrap _badge_ for counts. Dummy code:

```html
<div class="row">
  <div class="col-sm-4">
    <dl class="dl-horizontal">
      <dt>projects:</dt><dd>{{ number_of_projects }}</dd>
      <dt>flowcells:</dt><dd>{{ number_of_flowcells }}</dd>
      <dt>lanes:</dt><dd>{{ number_of_lanes }}</dd>
      <dt>samples:</dt><dd>{{ number_of_samples }}</dd>
    </dl>
  </div>
  <div class="col-sm-4">
    <h4>Filter by flowcell status</h4>
    <p>
      <button class="btn btn-default btn-sm" class="check-all" data-target=".fc-status-checkbox">Check all</button>
      <button class="btn btn-default btn-sm" class="check-none" data-target=".fc-status-checkbox">Check none</button>
    </p>
    {% for status, number in sorted(status_list.items()) %}
      <div class="checkbox">
        <label>
          <input class="fc-status-checkbox" type="checkbox" value="{{ status }}" checked>
          {{ status }} <span class="badge">{{ number }}</span>
        </label>
      </div>
    {% end %}
  </div>
  <div class="col-sm-4">
    <h4>Filter by bioinfo responsible</h4>
    <p>
      <button class="btn btn-default btn-sm" class="check-all" data-target=".bi-responsible-checkbox">Check all</button>
      <button class="btn btn-default btn-sm" class="check-none" data-target=".bi-responsible-checkbox">Check none</button>
    </p>
    {% for responsible, number in sorted(responsible_list.items()) %}
      <div class="checkbox">
        <label>
          <input class="bi-responsible-checkbox" type="checkbox" value="{{ responsible }}" checked>
          {{ responsible }} <span class="badge">{{ number }}</span>
        </label>
      </div>
    {% end %}
  </div>
</div>
```

(I also added example _check-all_ and _check-none_ buttons...